<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Dados</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Objetivo</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Quantidade</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDial" name="dialTotal">
                      <property name="maximum">
                       <number>100</number>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
                      <item>
                       <widget class="QSpinBox" name="Total">
                        <property name="frame">
                         <bool>true</bool>
                        </property>
                        <property name="minimum">
                         <number>1</number>
                        </property>
                        <property name="maximum">
                         <number>100</number>
                        </property>
                        <property name="value">
                         <number>30</number>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_8">
                        <property name="text">
                         <string>ml</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Dosaegem</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDial" name="dialDosagem"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0">
                      <item>
                       <widget class="QSpinBox" name="DosagemObjetivo">
                        <property name="minimum">
                         <number>1</number>
                        </property>
                        <property name="maximum">
                         <number>1000</number>
                        </property>
                        <property name="value">
                         <number>18</number>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_9">
                        <property name="text">
                         <string>mg/ml</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Propileno Glicol</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDial" name="dialPG"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
                      <item>
                       <widget class="QSpinBox" name="PercentualPG">
                        <property name="maximum">
                         <number>100</number>
                        </property>
                        <property name="value">
                         <number>60</number>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_14">
                        <property name="text">
                         <string>%</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Glicerina Vegetal</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QDial" name="dialVG"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,0">
                      <item>
                       <widget class="QSpinBox" name="PercentualVG">
                        <property name="maximum">
                         <number>100</number>
                        </property>
                        <property name="value">
                         <number>40</number>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="label_7">
                        <property name="text">
                         <string>%</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Dosagem NIC</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
               <item>
                <widget class="QSpinBox" name="DosagemNIC">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>mg/ml</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="sliderNic">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Resultados</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>PG:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ResPG"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>VG:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ResVG"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>NIC</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ResNIC"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>PercentualPG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialPG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PercentualVG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialVG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DosagemObjetivo</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialDosagem</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialPG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PercentualPG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialTotal</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Total</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderNic</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DosagemNIC</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Total</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialTotal</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialVG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PercentualVG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DosagemNIC</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderNic</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialDosagem</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DosagemObjetivo</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotTotalChanged()</slot>
  <slot>slotDosagemChanged()</slot>
  <slot>slotPGChanged()</slot>
  <slot>slotVGChanged()</slot>
  <slot>slotNICChanged()</slot>
 </slots>
</ui>
